_quarto.yml

```diff
@@ -9,7 +11,7 @@ website:
     position: below-navbar
   favicon: logo/logo.png
   title: "hubverse"
-  site-url: "https://hubverse-org.github.io"
+  site-url: "https://hubverse.io"
   site-path: "/hubverse-site/"
```

Comment:
re: https://github.com/hubverse-org/hubverse-site/pull/17/files#r2070270696

Yes, this was the change. This should change, too:

```suggestion
```

see https://quarto.org/docs/websites/website-navigation.html#pages-404

Reply:
Ah! Got it. Thanks!